### ISSUE TITLE
[Java] GuardLogStatement does not understand negative if conditions

### ISSUE THREAD

**Issue by nathanila** [pmd/pmd#1287]
**Affects PMD Version:** 
Eclipse PMD Plug-in 1.13.0.201806091459

**Rule:**
GuardLogStatement

**Description:**
A short-circuit if statement at the top of the method is not detected by GuardLogStatement and hence a PMD warning shows up on the later log statement.

**Code Sample demonstrating the issue:**

Problematic code:
```
private void log()
{
   if (!s_logger.isDebugEnabled())
      return;
   ...
   s_logger.debug(text);      // PMD warning shows up here
}
```

Working code:
```
private void log()
{
   if (s_logger.isDebugEnabled())
   {
      ...
      s_logger.debug(text);      // No PMD warning
   }
}
```

The "..." has identical code in it.

**Running PMD through:** *[CLI | Ant | Maven | Gradle | Designer | Other]*
Eclipse
